src/main/resources/database.sql

```diff
@@ -1,2 +1,21 @@
-create table MEMBER(id bigint auto_increment, email varchar(255), name varchar(255), password varchar(255), role varchar(10));
```

Comment:
I think it's better to add tracking column like `create_ymdt` and `last_mod_ymdt`.